docs/src/styles/mixins.js

```diff
@@ -0,0 +1,51 @@
+import { css } from 'styled-components';
```

Comment:
🎉 Noticed myself writing stacks and centering logic over and over again – so I made this "mixins" file to abstract them, much like I would do in Sass. This uses the `css` utility from `styled-components` and then these just get added into the template strings for individual styled components.